Framework/YYCache.xcodeproj/project.pbxproj

```diff
@@ -277,6 +277,7 @@
 		641EE7272240E51B00173FCB /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				"ARCHS[sdk=appletvos*]" = arm64;
```

Comment:
is this line needed?

Reply:
No, for now.
I expected this explicit preference helped when we built this project for M1 mac or Intel one. For iiponesimulator, it inherited parents project/settings but real devices were only for arm64. But this did not change them. (So, either with or without this has no difference for our purpose)